### PR TITLE
waitAfterScroll to protected

### PR DESCRIPTION
**Pull request by teunisnl**
also moved double click to selenium helper

### CODE REVIEW

File: src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java [hunk 1]
```diff
@@ -1409,6 +1407,13 @@ protected WebElement getElementToScrollTo(String place, String container) {
      */
     protected void scrollTo(WebElement element) {
         getSeleniumHelper().scrollTo(element);
+        waitAfterScroll();
+    }
+
+    /**
+     * Wait after the scroll if needed
+     */
+    protected void waitAfterScroll() {
```
Comment (fhoeben) [inline src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java]: make `waitAfterScroll` a parameter of this method, so subclasses can also access it.

Comment (teunisnl) [inline src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java]: is access by waitMilliSecondAfterScroll(int msToWait) not good enough ?

Comment (fhoeben) [inline src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java]: That is a setter, to change its value, it does not provide access to current value

**Comment by fhoeben** [comment]
I was about to merge this. 
A separate pull request for drag and drop would have been nice. For me, drag and drop, would ideally have an acceptance test. Would that be hard to add?

**Comment by teunisnl** [comment]
ill undo Drag and Drop.

html5 does not work [(issue report)](https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/3604)
and a working html4 example I cant seem to find. (cant access the website where its working on)

**Pull request merged**